### DISCUSSION TITLE
Contribute to projects

### DISCUSSION THREAD

**Discussion by onkar-1432** [ishuonly/ExpirationDateTracker#4] [General]
Hey! Just saw your Project on kwoc. Would love to contribute as well understand the project.

**Comment by ishuonly** [comment]
Hey! 
So first familiarize yourself with the project's overall objective and different pages and layers by reading the readme of the main branch, and then go through [Contribution Guidelines](https://github.com/ishuonly/ExpirationDateTracker/blob/main/CONTRIBUTION.md).

Tip: Checkout different branches for different functionalities of the project.
